procsim/biomass/level0_product_generator.py

```diff
@@ -408,8 +417,11 @@ def generate_output(self):
         self._hdr.product_type = self._output_type
         self._hdr.incomplete_l0_slice = False
         self._hdr.partial_l0_slice = False
-        acq = self._hdr.acquisitions[0]
-        acq.slice_frame_nr = None
+        
+        # Get slice number from middle of slice to deal with merged slices.
+        middle_position = self._hdr.begin_position + (self._hdr.end_position - self._hdr.begin_position) / 2
+        self._hdr.acquisitions[0].slice_frame_nr = self._get_slice_frame_nr(middle_position, constants.SLICE_GRID_SPACING)
```

Comment:
See previous comment

Reply:
Reverted in https://github.com/stcorp/procsim/pull/17/commits/672e56460cacbfd00ebf94ab26b392ad0791c5a1.